packages/react-components/src/components/RichTextEditor/Plugins/UpdateContentPlugin.tsx

```diff
@@ -101,3 +96,18 @@ export class UpdateContentPlugin implements EditorPlugin {
     this.onUpdate(UpdateEvent.Blur);
   };
 }
+
+const getImagesSourcesList = (contentModel?: ContentModelDocument): Array<string> => {
+  const imageSrcArray: Array<string> | undefined = [];
+  contentModel?.blocks.map((block) => {
+    if (block.blockType === 'Paragraph') {
```

Comment:
Are the images always nested in a paragraph block?

Reply:
Yes, everything should be in a paragraph block (as per previous discussions with RoosterJS team)